src/test/scripts/org/reaktivity/reaktor/test/k3po/ext/duplex/client.sent.option.flags.fragmentation/client.rpt

```diff
@@ -0,0 +1,32 @@
+#
+# Copyright 2016-2020 The Reaktivity Project
+#
+# The Reaktivity Project licenses this file to you under the Apache License,
+# version 2.0 (the "License"); you may not use this file except in compliance
+# with the License. You may obtain a copy of the License at:
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+
+connect "nukleus://streams/server#0"
+        option nukleus:transmission "duplex"
+        option nukleus:window 8192
+connected
+
+write option nukleus:flags ${nukleus:flags("init")}
```

Comment:
I realize this is the syntax we discussed, but when I look at it now it seems quite verbose.

Seems like it would be better to go with something slightly simpler, like:
```
write option nukleus:flags "init"
```
```
write option nukleus:flags "none"
```
```
write option nukleus:flags "fin"
```
```
write option nukleus:flags "auto"
```
and multiple values via
```
write option nukleus:flags "init" "fin"
```
almost the same as the EL function approach.